gui/src/components/CopyJobSpec.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Button from '@material-ui/core/Button';
```

Comment:
Can you run the linter over this PR and change the tabs to spaces 🙏

Reply:
Sure, but I'm getting lint errors:
```
error  Parsing error: Unexpected token import  null

✖ 1 problem
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chainlink@0.2.0 lint: `standard --fix --verbose | snazzy`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the chainlink@0.2.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Reply:
You may need to rebase the branch and update the packages with yarn